ai-server/app/router/recommendation_router.py

```
@@ -14,10 +14,32 @@
+        response = requests.get(url, timeout=10)
+        
+        if response.status_code == 200:
+            hidden_projects = response.json()
+            logger.info(f"🚫 사용자 {user_id} 숨김 프로젝트: {len(hidden_projects)}개")
+            return hidden_projects
+        else:
+            logger.warning(f"⚠️ 숨김 프로젝트 조회 실패 (HTTP {response.status_code})")
+            
+    except requests.RequestException as e:
+        logger.warning(f"⚠️ Spring Boot API 연결 실패: {e}")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)